### PR TITLE
Final-final GFS v16 updates / restart reproducibility bugfixes

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

This PR updates the submodule pointer for fv3atm (and ccpp-physics underneath)

-  for the final-final (!) GFS v16 physics updates
- to obtain restart reproducibility for uncoupled and coupled runs

These code changes were contributed by @SMoorthi-emc, @junwang-noaa and @yangfanglin.

Additional changes:
- create separate GFS v16 beta stochastic physics test (with update stochastic physics settings as in `fv3_ccpp_stochy`)
- add GFS v16 beta restart test
- run GFS v16 tests also on wcoss_dell_p3 and wcoss_cray (contributed by @junwang-noaa)

### Issue(s) addressed

https://github.com/ufs-community/ufs-weather-model/issues/272
https://github.com/ufs-community/ufs-weather-model/issues/239 (potentially - but need to check)

## Testing

Regression tests were run on all tie-1 platforms, logs updated in the PR.

The changes in this PR do not change the results for the existing tests, other than:
- the surface restart files for all tests, because of the additional `tiice` field being written to it
- the GFS v16 beta tests that were changed or added (turn off stochastic physics in standard GFS v16 beta tests, add restart and stochastic test)

The new baseline date tag is 20201214.

## Dependencies

https://github.com/NCAR/ccpp-physics/pull/531
https://github.com/NOAA-EMC/fv3atm/pull/212
https://github.com/ufs-community/ufs-weather-model/pull/325

### CODE REVIEW

File: tests/rt.conf [hunk 1]
```diff
@@ -120,6 +120,12 @@ RUN     | fv3_ccpp_gfs_v15p2
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | hera.intel     | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | orion.intel    | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | cheyenne.intel | fv3         |
+RUN     | fv3_ccpp_gfs_v16beta_restart                                                                                                   | standard    | hera.intel     |             | fv3_ccpp_gfs_v16beta
```
Comment (DusanJovic-NOAA) [inline tests/rt.conf:129]: Why don't we run fv3_ccpp_gfs_v16beta* tests on wcoss?

Comment (climbfuji) [reply tests/rt.conf:129]: No idea, we never did.

Comment (climbfuji) [reply tests/rt.conf:129]: These tests are now run on both wcoss platforms.

**Comment by climbfuji** [comment]
This PR is ready for final review and merging.

**Review by junwang-noaa — APPROVED** [review]

**Review by DusanJovic-NOAA — APPROVED** [review]

**Pull request merged**